stripe-attestation/src/test/java/com/stripe/attestation/IntegrityStandardRequestManagerTest.kt

```diff
@@ -0,0 +1,102 @@
+package com.stripe.attestation
+
+import android.app.Activity
+import com.google.android.gms.tasks.Task
+import com.google.android.gms.tasks.Tasks
+import com.google.android.play.core.integrity.StandardIntegrityManager
+import com.google.android.play.core.integrity.StandardIntegrityManager.StandardIntegrityToken
+import com.google.android.play.core.integrity.StandardIntegrityManager.StandardIntegrityTokenProvider
+import com.google.android.play.core.integrity.StandardIntegrityManager.StandardIntegrityTokenRequest
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.test.StandardTestDispatcher
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
+import org.junit.Before
+import kotlin.test.Test
+
+class IntegrityStandardRequestManagerTest {
+
+    private val testDispatcher = StandardTestDispatcher()
+
+    @Before
+    fun setUp() {
+        Dispatchers.setMain(testDispatcher)
```

Comment:
Do we need to reset this after tests?

Reply:
https://github.com/stripe/stripe-android/pull/9464/commits/eca40c2ebf97545195ada599df0395c7732d369d